File: backend/TaskHero/TaskHero.Domain/Requests/RequestType.cs
using System.ComponentModel;

namespace TaskHero.Domain.Requests;

public enum RequestType
{
    [Description("Moving & Lifting")]
    MovingAndLifting,
    
    [Description("Gardening")]
    Gardening,
    
    [Description("Pet Care")]
    PetCare,
    
    [Description("Grocery Pickup")]
    GroceryPickup,
    
    [Description("Handyman")]
    Handyman
}

File: frontend/src/enums/request-type.ts
export const REQUEST_TYPES = {
  MOVING: 0,
  GARDENING: 1,
  PET_CARE: 2,
  GROCERY: 3,
  HANDYMAN: 4,
} as const

export type RequestType = (typeof REQUEST_TYPES)[keyof typeof REQUEST_TYPES]
